### PR TITLE
Personio client prepare

### PR DESCRIPTION
**Pull request by ArktinenSieni**

### CODE REVIEW

File: personio-client/src/Personio/Types.hs [hunk 1]
```diff
@@ -136,7 +136,7 @@ data Employee = Employee
     , _employeeFlowdock       :: !(Maybe FD.UserId)
     , _employeeStatus         :: !Status
     , _employeeHRNumber       :: !(Maybe Int)
-    , _employeeEmploymentType :: !EmploymentType
+    , _employeeEmploymentType :: !(Maybe EmploymentType)
```
Comment (phadej) [inline personio-client/src/Personio/Types.hs]: do we have a check if employment type is missing?

Comment (ArktinenSieni) [inline personio-client/src/Personio/Types.hs]: At least there is a validation for it at row 516.

Comment (phadej) [inline personio-client/src/Personio/Types.hs]: :+1:

**Comment by phadej** [comment]
Merge when green.

**Comment by phadej** [comment]
red

**Pull request merged**